### PR TITLE
Working on add to scrape products save data from each page and save i…

### PR DESCRIPTION
**Pull request by harmeshverma01**
…n json format

### CODE REVIEW

File: util/utils.py [hunk 1]
```diff
@@ -36,12 +33,10 @@ def product_details(divs, list_of_product):
         
         product = {
             'name' : product_name.text,
-            'rating' : product_rating.text,
+            # 'rating' : product_rating.text,
```
Comment (akashSharma135) [inline util/utils.py:36]: Why rating field is commented?
We need to store rating as well.

File: util/utils.py [hunk 2]
```diff
@@ -55,9 +50,11 @@ def html_content(url):
 
 def get_next_page(url):
     soup = html_content(url)
+    print(soup)
     nextpage = soup.find('a', {'class' : '_1LKTO3'})
+    print("SHJNJN: ", nextpage)
```
Comment (akashSharma135) [inline util/utils.py:55]: remove it

File: util/utils.py [hunk 3]
```diff
@@ -55,9 +50,11 @@ def html_content(url):
 
 def get_next_page(url):
     soup = html_content(url)
+    print(soup)
```
Comment (akashSharma135) [inline util/utils.py:53]: remove it

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,4 @@ venv
 __pycache__/
 db.sqlite3
```
Comment (akashSharma135) [inline .gitignore:3]: Are we using db.sqlite?

File: .gitignore [hunk 5]
```diff
@@ -2,4 +2,4 @@ venv
...
 .vscode
-hr.py
\ No newline at end of file
+data.json
```
Comment (akashSharma135) [inline .gitignore:5]: json file is still pushed on git.

File: util/utils.py (hunk 1 above)
Comment (harmeshverma01) [reply util/utils.py:36]: fixed

File: util/utils.py (hunk 2 above)
Comment (harmeshverma01) [reply util/utils.py:55]: fixed

File: util/utils.py (hunk 3 above)
Comment (harmeshverma01) [reply util/utils.py:53]: fixed

File: .gitignore (hunk 4 above)
Comment (harmeshverma01) [reply .gitignore:3]: fixed

File: .gitignore (hunk 5 above)
Comment (harmeshverma01) [reply .gitignore:5]: fixed

**Pull request merged**